### ISSUE TITLE
Is this package planned to be actively maintained?

### ISSUE THREAD

**Issue by kaushalmodi** [russross/blackfriday#430]
.. or this package actively maintained. 

I am just looking all the old issues, and new issues being opened up about once every week or every other week. 

But those issues seem to land up in a blackhole somewhere.. I have a handful issues open many months ago that I have opened, that haven't been resolved, or even responded to.

Looking at https://github.com/russross/blackfriday/pulse/monthly is depressing.. almost no project activity.

Are you guys looking for someone to continue this project?

**Comment by shurcooL** [comment]
Have you read https://github.com/russross/blackfriday#versions? Most (all?) new development is happening on `v2` branch, and https://github.com/russross/blackfriday/pulse/monthly doesn't seem to capture that.

**Comment by kaushalmodi** [comment]
> Have you read

Yes, I have, but you can see below that there's no serious activity on the project in last 5-6 months on any branch, and number of issues are increasing gradually, and no one is responding if that issue will ever get fixed (*or not*).

![image](https://user-images.githubusercontent.com/3578197/35492561-9f7ec148-047b-11e8-8053-9ac59efefc0c.png)

In addition, there are PR's getting submitted, but no one ever gives any feedback (*good? bad.. won't merge? ok.. here's what you need to fix before merge?*).. nothing. I am especially interested in https://github.com/russross/blackfriday/pull/412 (that PR is also based off v2).

---

> new development is happening on v2 branch, and I'm not sure if https://github.com/russross/blackfriday/pulse/monthly captures that

It does track **all** branches:

![image](https://user-images.githubusercontent.com/3578197/35492652-27f907b8-047c-11e8-8798-3fe9a2c3e36f.png)

---

I am bringing this up, because I love the `hugo` project and that project depends on this one. If `blackfriday` team is looking for folks to help out, probably giving a shout out on https://discourse.gohugo.io/ would help. I wish I can add some contribution to this project too, but I am not a Go developer.
